charts/sentry/templates/sentry/worker/deployment-sentry-worker-events.yaml

```diff
@@ -76,10 +76,18 @@ spec:
       - name: {{ .Chart.Name }}-worker
         image: "{{ template "sentry.image" . }}"
         imagePullPolicy: {{ default "IfNotPresent" .Values.images.sentry.pullPolicy }}
-        command: ["sentry"]
+        command: ["/bin/bash", "-c"]
```

Comment:
It's better to use init container and emptyDir. Нou will get the same result without changing the main container command

Reply:
What commands should I run in the init container?

Reply:
Same commands, but in different container and mount emptyDir for `/usr/local/share/ca-certificates/`. Like [init container in relay](https://github.com/sentry-kubernetes/charts/blob/develop/charts/sentry/templates/relay/deployment-relay.yaml#L82)

Reply:
@adonskoy could you create MR with you idea?

Reply:
@adonskoy there is no point in adding certificates to python storage using init container since python certificate storage in the main container is dynamic and can change from version to version

Reply:
I also need to install additional python packages. They cannot be installed via init container